src/firecracker/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "firecracker"
 version = "0.21.0"
 authors = ["Amazon Firecracker team <firecracker-devel@amazon.com>"]
 edition = "2018"
+build = "../../build.rs"
```

Comment:
Shouldn't we do this for `jailer` too?

Reply:
This comment is outdated. In the current state of the PR, it is not possible, since both of these versions would be identical and would point to the same firecracker version number